client/src/views/WaterTrackingView.jsx

```diff
@@ -44,6 +44,13 @@ export const WaterTrackingView = ({ currentIntake, setCurrentIntake }) => {
 
   const percentage = (currentIntake * 100)/WATER_GOAL;
 
+  const isDecrementBottonDisabled = () => {
+    if (currentIntake > 0) {
+    return false;
+    }
+    return true;
+  };
+
```

Comment:
1. This could be simplified to a computed property, no need to create a function as `currentIntake < 0` is a boolean. Also I reversed here the property in order to make it more readable than `!(currentIntake > 0)` 

```
const isDecrementBottonDisabled = currentIntake < 0;
```

2. Also take into account that there is a type `Botton` as opposed to `Button`

3. Please adress this bug in another PR